blog/2021/12/blogpost.md

```diff
@@ -0,0 +1,261 @@
+
```

Comment:
Maybe use a more meaningful file name?  Otherwise the URL would be https://julialang.org/blog/2021/12/blogpost/ which isn't much useful

Reply:
Thanks! Makes sense - forgot there can be multiple .md files in that dir

Reply:
Good catch @giordano